### ISSUE TITLE
topaz tutorial error

### ISSUE THREAD

**Issue by jliu321** [tbepler/topaz#31]
Our lab just installed Topaz via SBgrid and I just start to learn how to use it. I am following the tutorial data and found that it gave me error on the train step. I am not sure what cause it (I am just copying and pasting things).Thank you in advance for your help!
 Below is the output message: # Loading model: resnet8
# Model parameters: units=32, dropout=0.0, bn=on
# Receptive field: 71
# Using device=0 with cuda=True
# Loaded 20 training micrographs with 1000 labeled particles
# Loaded 10 test micrographs with 500 labeled particles
# source	split	p_observed	num_positive_regions	total_regions
# 0	train	0.00654	29000	4435200
# 0	test	0.00654	14500	2217600
# Specified expected number of particle per micrograph = 300.0
# With radius = 3
# Setting pi = 0.03923160173160173
# minibatch_size=256, epoch_size=5000, num_epochs=10
THCudaCheck FAIL file=/opt/conda/conda-bld/pytorch_1544174967633/work/aten/src/THC/THCGeneral.cpp line=405 error=11 : invalid argument

**Comment by tbepler** [comment]
Is something else running on the same GPU as Topaz? This is a cuda related error that sometimes occurs when there is not enough available RAM on the GPU.

**Comment by jliu321** [comment]
Thank you very much. You are right. I checked the GPU usage, someone else
is using the GPUs. My work is running on one of them, and now it is done.

Thanks.

Tristan Bepler <notifications@github.com> 于2019年7月5日周五 下午2:26写道：

**Comment by tbepler** [comment]
I'm going to close this issue since it sounds like the problem is resolved.
